src/main/kotlin/uk/gov/justice/digital/hmpps/findandreferanintervention/jpa/entity/DeliveryMethodSetting.kt

```diff
@@ -20,8 +23,9 @@ open class DeliveryMethodSetting(
   open var id: UUID,
 
   @NotNull
-  @Column(name = "delivery_method_id")
-  open var deliveryMethodId: UUID,
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "intervention_id", referencedColumnName = "id")
+  open var intervention: InterventionCatalogue?,
```

Comment:
Catalogue shouldn't be null
```suggestion
  open var intervention: InterventionCatalogue,
```

Reply:
Agreed, just realised I had updated the annotation but not the variable. Will do this in the next commit.